mysql/datadog_checks/mysql/mysql.py

```diff
@@ -913,9 +936,10 @@ def _get_replica_status(self, db, above_560, nonblocking):
     def _are_values_numeric(cls, array):
         return all(v.isdigit() for v in array)
 
-    def _get_variable_enabled(self, results, var):
+    @staticmethod
+    def _get_variable_enabled(results, var):
```

Comment:
Just noticing this, but we have `is_affirmative` in the base check for this use case: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/config.py#L6

Reply:
oh nice, tyvm!

Reply:
In the future, could you remove this function and the related ones altogether, and just use `is_affirmative` when getting the config values?

Reply:
Yeah, we'll be going back to improve some old things and make improvements such as: https://github.com/DataDog/integrations-core/pull/12343#discussion_r892830040

Reply:
Awesome, thanks!